src/components/ApprovalWorkflowSection.tsx

```diff
@@ -0,0 +1,103 @@
+import React, {useCallback} from 'react';
+import {View} from 'react-native';
+import useLocalize from '@hooks/useLocalize';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import Navigation from '@libs/Navigation/Navigation';
+import ROUTES from '@src/ROUTES';
+import type ApprovalWorkflow from '@src/types/onyx/ApprovalWorkflow';
+import Icon from './Icon';
+import * as Expensicons from './Icon/Expensicons';
+import MenuItem from './MenuItem';
+import PressableWithoutFeedback from './Pressable/PressableWithoutFeedback';
+import Text from './Text';
+
+type ApprovalWorkflowSectionProps = {
+    approvalWorkflow: ApprovalWorkflow;
```

Comment:
Please add `/** */` style comments to each prop

Reply:
Just added, please take a quick look if they're okay :)